enclave-runtime/rust-toolchain.toml

```diff
@@ -1,4 +1,4 @@
 [toolchain]
-channel = "nightly-2021-11-10"
+channel = "nightly-2022-03-10"
```

Comment:
Updated toolchain to the latest nightly version

Reply:
above 03-10 is not possible? Nightly of march does not seem very up to date. Not that it is required.

Reply:
I don't think I understand? The toolchain in both files is now `2022-03-10`? It's the version just one above the one teaclave uses and works for us.

Reply:
It's fine. I was just wondering because you wrote "latest". :)

Reply:
Yes, you're right, that is incorrect 😄 Updated the PR description accordingly.